src/machine.rs

```diff
@@ -19,8 +20,9 @@ use matrix_sdk_crypto::{
     types::RoomKeyBackupInfo,
     CryptoStoreError, EncryptionSyncChanges, GossippedSecret,
 };
+use serde::{ser::SerializeSeq, Serialize, Serializer};
 use serde_json::json;
-use serde_wasm_bindgen;
+use serde_wasm_bindgen::{self};
```

Comment:
nit: this is redundant, as it's equivalent to the previous form. Doesn't Clippy complain about this?

Reply:
Weirdly, Clippy doesn't complain, but I will fix it. No idea how it appeared.